lib/rules/comma-dangle.js

```diff
@@ -128,10 +132,14 @@ module.exports = function(context) {
         var sourceCode = context.getSourceCode();
         var trailingToken = sourceCode.getTokenAfter(lastItem);
         if (trailingToken.value !== ",") {
-            context.report(
-                lastItem,
-                lastItem.loc.end,
-                MISSING_MESSAGE);
+            context.report({
+                node: lastItem,
+                loc: lastItem.loc.end,
+                message: MISSING_MESSAGE,
+                fix: function(fixer) {
+                    return fixer.insertTextAfter(lastItem, ",");
```

Comment:
1. `comma-spacing` is not automatically fixable yet
2. It could then cause the `max-len` rule to error, which could also happen if the fix of the `comma-dangle` rule is applied. So you could even get 2 errors by fixing one.

I think we just need a rule on how we want to deal with such issues in general.